docs/components/conditional_rendering.md

```diff
@@ -267,3 +267,57 @@ def cond_style_example():
         ),
     )
 ```
+## Multiple Conditional Statements 
+
+The `rx.match` component in Reflex provides a powerful alternative to`rx.cond` for handling multiple conditional statements and structural pattern matching. This component allows you to handle multiple conditions and their associated components in a cleaner and more readable way compared to nested `rx.cond` structures.
+
+```python demo exec
+from typing import List
+
+import reflex as rx
+
+
+class MatchState(rx.State):
+    cat_breed: str = ""
+    animal_options: List[str] = [
+        "persian",
+        "siamese",
+        "maine coon",
+        "ragdoll",
+        "pug",
+        "corgi",
+    ]
+
+
+def match_demo():
+    return rx.flex(
+        rx.match(
+            MatchState.cat_breed,
+            ("persian", rx.text("Persian cat selected.")),
+            ("siamese", rx.text("Siamese cat selected.")),
+            (
+                "maine coon",
+                rx.text("Maine Coon cat selected."),
+            ),
+            ("ragdoll", rx.text("Ragdoll cat selected.")),
+            rx.text("Unknown cat breed selected."),
+        ),
+        rx.select.root(
```

Comment:
We have a high level select that we should use here

Reply:
Fixed